app/src/main/java/com/teamagam/gimelgimel/app/common/logging/Log4jDiskLogger.java

```diff
@@ -0,0 +1,221 @@
+package com.teamagam.gimelgimel.app.common.logging;
+
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+
+import java.io.File;
+
+import de.mindpipe.android.logging.log4j.LogConfigurator;
+
+
+class Log4jDiskLogger extends BaseLifecycleLogger {
+
+    private static final String VERBOSITY_USER_INTERACTION = "USER_INTERACTION";
+    private static final String VERBOSITY_LIFECYCLE = "LIFECYCLE";
+
+    private Logger mLogger;
```

Comment:
Since we also use the Logger class, It might be better to use the full package-named name for this Logger to avoid misinformation.